e2e/Dockerfile

```
@@ -0,0 +1,4 @@
+FROM docker.io/fedora:36
+
+LABEL org.opencontainers.image.source https://github.com/k8snetworkplumbingwg/multi-networkpolicy-iptables
```

This can help a new contributor to understand where the test image comes from.

```suggestion
LABEL org.opencontainers.image.source https://github.com/k8snetworkplumbingwg/multi-networkpolicy-iptables
LABEL org.opencontainers.image.base.name ghcr.io/k8snetworkplumbingwg/multi-networkpolicy-iptables:e2e-test
```

---

changed.